# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/sessionstore/test/browser.ini
# Commit: f155dce29134
# Full Hash: f155dce2913463e41a589af8eec1dbc24c42aac3
# Author: Kashav Madan <kshvmdn@gmail.com>
# Date: 2021-06-16 09:46:08
# Regressor Bug: 1701337
# File Overlap Count: 1
# Description:
#   Bug 1701337 - Respect browser.sessionstore.debug.no_auto_updates in native flushers, r=nika
#   
#   We should not be doing any timed flushes if the pref is set to true.
#   
#   See https://searchfox.org/mozilla-central/rev/d97dca5907e2a0a77a2435b24ef980268cd3c4fe/browser/components/sessionstore/ContentSessionStore.jsm#402
# ==============================================================================

diff -r 14d2731278cb -r f155dce29134 browser/components/sessionstore/test/browser.ini
--- a/browser/components/sessionstore/test/browser.ini	Wed Jun 16 00:31:15 2021 +0000
+++ b/browser/components/sessionstore/test/browser.ini	Wed Jun 16 00:37:53 2021 +0000
@@ -287,8 +287,6 @@
 [browser_1284886_suspend_tab.js]
 skip-if = !e10s
 [browser_async_window_flushing.js]
-skip-if =
-  fission && (os != "linux" || asan)  # Bug 1713900 - new Fission platform triage
 [browser_focus_after_restore.js]
 [browser_forget_async_closings.js]
 [browser_movePendingTabToNewWindow.js]